frameworks/digital-outcomes-and-specialists/questions/services/serviceName.yml

```diff
@@ -0,0 +1,17 @@
+question: What is the name of studio?
+type: text
+hint: A studio is the room in which the participant(s) and moderator(s) carry out testing. The name will help people buyers refer to the right studio when communicating with you.
```

Comment:
This should be "people" or "buyers", but not "people buyers".

Unless we're doing a framework for human trafficking? :trollface: